sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/LogicalQueryStage.scala

```diff
@@ -53,8 +54,14 @@ case class LogicalQueryStage(
   override def computeStats(): Statistics = {
     // TODO this is not accurate when there is other physical nodes above QueryStageExec.
```

Comment:
According to the PR description, we have still issues for the other physical nodes?
> Fix data issue. OptimizeOneRowPlan will use stats to remove Aggregate:

Reply:
I've checked other physical nodes and none of them seem to have this issue.

Reply:
Then, shall we remove this comment?

Reply:
I think keep it, new physical nodes may be added in the future.

Reply:
this comment should be still valid for aggregation with non-empty query stage as this pr only changes the empty case.